### PR TITLE
Add support for more environment variables and improve README.md

### PR DESCRIPTION
**Pull request by chipironcin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,6 +264,40 @@ $ docker run -d --name wordpress -p 80:80 -p 443:443 \
...
 
+### SMTP Configuration
+
+To configure WordPress to send email using SMTP you can set the following environment variables:
+- `WORDPRESS_SMTP_HOST`: Host for outgoing SMTP email.
+- `WORDPRESS_SMTP_PORT`: Port for outgoing SMTP email.
+- `WORDPRESS_SMTP_USER`: User of SMTP used for authentication (likely email).
+- `WORDPRESS_SMTP_PASSWORD`: Password for SMTP.
+- `WORDPRESS_SMTP_USERNAME`: User name for SMTP emails.
+- `WORDPRESS_SMTP_PROTOCOL`: Secure connection protocol to use for SMTP [tls, ssl, none].
```
Comment (sameersbn) [inline README.md]: typo: `WORDPRESS_SMTP_*` -> `SMTP_*`

File: README.md [hunk 2]
```diff
@@ -261,6 +264,40 @@ $ docker run -d --name wordpress -p 80:80 -p 443:443 \
...
+```
+  application:
+    image: bitnami/wordpress:latest
+    ports:
+      - 80:80
+    environment:
+      - SMTP_HOST=smtp.gmail.com
+      - SMTP_PORT=587
+      - SMTP_USER=your_email@gmail.com
+      - SMTP_PASSWORD=your_password
```
Comment (sameersbn) [inline README.md]: should the `SMTP_USERNAME` parameter be added to this snippet?

File: rootfs/app-entrypoint.sh [hunk 3]
```diff
@@ -13,14 +15,21 @@ function initialize {
...
+export WORDPRESS_FIRST_NAME=${WORDPRESS_FIRST_NAME:-"FirstName"}
+export WORDPRESS_LAST_NAME=${WORDPRESS_LAST_NAME:-"LastName"}
+export WORDPRESS_BLOG_NAME=${WORDPRESS_BLOG_NAME:-"User's Blog!"}
 export MARIADB_USER=${MARIADB_USER:-"root"}
 export MARIADB_HOST=${MARIADB_HOST:-"mariadb"}
 export MARIADB_PORT=${MARIADB_PORT:-"3306"}
 
-if [[ "$1" == "harpoon" && "$2" == "start" ]]; then
+
+
```
Comment (sameersbn) [inline rootfs/app-entrypoint.sh]: Please remove these extra new line characters.

**Comment by chipironcin** [comment]
Thanks for the review @sameersbn 
Lemme fix them ;)

File: README.md (hunk 2 above)
Comment (chipironcin) [inline README.md]: Not necessary, its optional

**Pull request merged**